From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Wed, 26 Aug 2015 13:17:17 +0100
Subject: configure.ac version

autoconf gets annoyed if configure.ac does not contain a literal
version argument in AC_INIT.  The upstream code tries to figure out
the version from the directory name and include it in.  This has a few problems:

* The code to do this is in Makefile, and is overwritten by autotools.
* The code doesn't work in Debian because the directory name does
  not necessarily encode the version.
* The file containing the version is not present in the upstream sources,
  and must be created by running "make", and autoconf fails if it is not
  found.
* The Makefile in the sources uses a syntactic extension not supported
  by GNU make, which errors out with "missing separator".

This patch passes a literal version number to AC_INIT.

BE SURE TO UPDATE FOR A NEW UPSTREAM VERSION!
---
 autocrap/configure.ac | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/autocrap/configure.ac b/autocrap/configure.ac
index 24e03fd..f7f8b8a 100644
--- a/autocrap/configure.ac
+++ b/autocrap/configure.ac
@@ -1,5 +1,4 @@
-m4_define([SCRYPT_VERS],m4_include([scrypt-version]))
-AC_INIT([scrypt],SCRYPT_VERS())
+AC_INIT([scrypt],[1.2.0])
 AC_CONFIG_AUX_DIR([config.aux])
 AM_INIT_AUTOMAKE()
 AC_PROG_CC
